### ISSUE TITLE
being detected as a trojan

### ISSUE THREAD

**Issue by iowesty** [sakura-ryoko/litematica#44]
Incompatible mods found!
net.fabricmc.loader.impl.FormattedException: net.fabricmc.loader.impl.discovery.ModResolutionException: Mod discovery failed!
	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:63)
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:198)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:146)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: net.fabricmc.loader.impl.discovery.ModResolutionException: Mod discovery failed!
	at net.fabricmc.loader.impl.discovery.ModDiscoverer.lambda$discoverMods$1(ModDiscoverer.java:160)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.discovery.ModDiscoverer.discoverMods(ModDiscoverer.java:160)
	at net.fabricmc.loader.impl.FabricLoaderImpl.setup(FabricLoaderImpl.java:216)
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:193)
	... 3 more
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: Error analyzing [C:\Software\ATLauncher\instances\VanilVaniltypeshit\mods\litematica-fabric-1.21-0.18.999-sakura.26.jar]: java.io.FileNotFoundException: C:\Software\ATLauncher\instances\VanilVaniltypeshit\mods\litematica-fabric-1.21-0.18.999-sakura.26.jar (Operation did not complete successfully because the file contains a virus or potentially unwanted software)
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at java.base/java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:540)
	at java.base/java.util.concurrent.ForkJoinTask.reportExecutionException(ForkJoinTask.java:580)
	at java.base/java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:993)
	at net.fabricmc.loader.impl.discovery.ModDiscoverer.discoverMods(ModDiscoverer.java:157)
	... 5 more
Caused by: java.lang.RuntimeException: Error analyzing [C:\Software\ATLauncher\instances\VanilVaniltypeshit\mods\litematica-fabric-1.21-0.18.999-sakura.26.jar]: java.io.FileNotFoundException: C:\Software\ATLauncher\instances\VanilVaniltypeshit\mods\litematica-fabric-1.21-0.18.999-sakura.26.jar (Operation did not complete successfully because the file contains a virus or potentially unwanted software)
	at net.fabricmc.loader.impl.discovery.ModDiscoverer$ModScanTask.compute(ModDiscoverer.java:305)
	at net.fabricmc.loader.impl.discovery.ModDiscoverer$ModScanTask.compute(ModDiscoverer.java:254)
	at java.base/java.util.concurrent.RecursiveTask.exec(RecursiveTask.java:110)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
Caused by: java.io.FileNotFoundException: C:\Software\ATLauncher\instances\VanilVaniltypeshit\mods\litematica-fabric-1.21-0.18.999-sakura.26.jar (Operation did not complete successfully because the file contains a virus or potentially unwanted software)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at java.base/java.util.zip.ZipFile$Source.<init>(ZipFile.java:1492)
	at java.base/java.util.zip.ZipFile$Source.get(ZipFile.java:1458)
	at java.base/java.util.zip.ZipFile$CleanableResource.<init>(ZipFile.java:724)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:251)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:180)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:194)
	at net.fabricmc.loader.impl.discovery.ModDiscoverer$ModScanTask.computeJarFile(ModDiscoverer.java:326)
	at net.fabricmc.loader.impl.discovery.ModDiscoverer$ModScanTask.compute(ModDiscoverer.java:295)
	... 7 more
![image](https://github.com/user-attachments/assets/57ba9a7d-848e-495b-a7ee-40b1ae3ab063)

**Comment by Inksday** [comment]
I also got this message,  I'm not going to say whether to trust it or not but apparently that specific trojan is a known false positive on minecraft mods.  Anyway I also uploaded the jar to virustotal and it was all clear.  I went ahead and gave windows permission to allow the file and used it for some time.  I made sure to scan my system with defender, malwarebytes, etc after the fact and it seems fine. 

At the end of the day use your own judgement and use at your own risk.
